demo/src/App.jsx

```diff
@@ -207,6 +204,23 @@ class App extends Component {
             <Section size="md">
               <Table stackAt="max-md" data={tableData} />
             </Section>
+            <Pagination.Wrapper ariaLabel="Pagination">
+              <Pagination.Item>
+                <Pagination.Link togglePreviousNext="previous" to="/" disabled>Prev</Pagination.Link>
```

Comment:
just `previous` without a value will be suffice here.  Same patter for `next`

Reply:
done